requirements.txt

```diff
@@ -1,3 +1,2 @@
-mock==1.3.0
 requests==2.20.0
```

Comment:
right now `requirements.txt` is just for people developing new features `aws_requests_auth` - it is not involved in the `setup.py` / package publishing component of the repo. So, https://github.com/DavidMuller/aws-requests-auth/pull/56 ("Prep for wheel support on pypi") excludes this change